AGPs/AGP-ANSP-X.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Retainer for Ongoing Aragon Network Security Provider
+Author: maurelian
+Status: Stage III
+Track: Finance
+Created: 2019-04-10
+---
+
+### Summary
+
+We propose to continue working with the Aragon One team on a retainer basis for a 1 year period. 
+
+## Address of the transfer recipient
+
+0xF0001193a7919B14417c038604846D7b3F8F4BC3
+
+## Amount of the transfer
+
+140,000 DAI
+20,000 ANT
+
+
+## Number and frequency of transfers if recurring (enter “1” if only one payment will be made)
+
+4 transfers, payable every 3 months starting after ANV-02 has completed.
```

Comment:
No, that amount 4 times.

* This is only 2/3 of the first AGP period's expenditure.
* Considering the amount of value and investment in the system (now and in the future), ~640k USD/yr is reasonable for securing it.
* Pricing isn't as simple as PPL * fee * audit weeks. We envision much engagement outside of that to support the entire community in improving security. This could include secure development trainings with flock teams, development of specific secure development guidelines, and other items. 

But I'm also happy to discuss adjustments that make this work for everyone.